general-superstaq/general_superstaq/superstaq_client.py

```diff
@@ -213,7 +227,49 @@ def get_job(self, job_id: str) -> Dict[str, str]:
         Raises:
             SuperstaqServerException: For other API call failures.
         """
-        return self.get_request(f"/job/{job_id}")
+        warnings.warn(
+            "Starting with client-superstaq v0.4.20, the `get_job` method and "
+            "endpoint won't exist. Call `fetch_jobs` instead.",
+            DeprecationWarning,
+            stacklevel=2,
+        )
+        return self.fetch_jobs([job_id])[0]
+
+    def fetch_jobs(
+        self,
+        job_ids: List[str],
+        tags: Optional[List[str]] = None,
+        targets: Optional[List[str]] = None,
```

Comment:
should we leave these out in anticipation of https://github.com/Infleqtion/server-superstaq/issues/2676?

Reply:
Depends on how long https://github.com/Infleqtion/server-superstaq/issues/2676 will take. This PR is a customer request so the sooner it gets in the better. 

We can get this in and update it as part of https://github.com/Infleqtion/server-superstaq/issues/2676. As you said it is part of the private `_SuperstaqClient` so we can easily update later.

Reply:
Removed `tags` and `targets` args.